src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -860,7 +860,10 @@ public static Path createTempFile(String prefix,
      * file-attributes} to set atomically when creating the directory. Each
      * attribute is identified by its {@link FileAttribute#name name}. If more
      * than one attribute of the same name is included in the array then all but
-     * the last occurrence is ignored.
+     * the last occurrence is ignored. When no file attributes are specified,
+     * then the resulting directory may have more restrictive access
+     * permissions to (non-temporary) directories created by the
```

Comment:
I re-read the text again and I think it would be better to drop "non-temporary" from this sentence. It would read better without it, and avoids introducing a new term.

Reply:
I was actually unsure of that also; will change.

Reply:
> it would be better to drop "non-temporary"

So changed in b99cf8f and CSR updated to match.